utils/data.js

```diff
@@ -75,7 +75,22 @@ export async function dataTransform(id) {
         };
       }
     });
-
+    
+    const consList = {};
+    dataParse.map((item, index) => {
+      if (consList[item[0]]) {
+          return;
+      } else {
+          if (item[0] == 'district_name') return;
+          consList[item[0]] = [
```

Comment:
Is this working? I don't see the return for the map function.

Reply:
![Screenshot from 2022-09-30 12-36-17](https://user-images.githubusercontent.com/55639487/193211338-e9d80e61-d872-4e1a-8d85-b96a8ff72e31.png)

getting the consList, here adding the district in else statement.